components/prism-fsharp.js

```diff
@@ -0,0 +1,16 @@
+Prism.languages.fsharp = Prism.languages.extend('clike', {
+	'comment': [
+		{
+			pattern: /(^|[^\\])\(\*[\w\W]*?\*\)/g,
+			lookbehind: true
+		},
+		{
+			pattern: /(^|[^\\:])\/\/.*?(\r?\n|$)/g,
```

Comment:
The last group matches the pattern found in prism-clike.js for single line comments
F# uses the same syntax for single line comments, but the comment property had to copied across as F# uses (* multi-line comment*) instead of the usual C /* multi-line comment */